docs/contributing/development-environment.md

```diff
@@ -76,3 +76,13 @@ information and steps below to set up a local development environment for Baby B
 
 Open [http://127.0.0.1:8000](http://127.0.0.1:8000) and log in with the default
 username and password (`admin`/`admin`).
+
+## Alternative: https://devenv.sh/
```

Comment:
This is actually smooth enough that I'm inclined to make it the primary development environment method... is devenv the only direct dependency?

Reply:
That's the beauty of devenv: it's the only dependency you need to install since it bootstraps everything else. Docker isn't a requirement. 

However, on windows, WSL2 or docker is a requirement. I actually made this PR with WSL2, but I also tested the vscode devcontainer, and GitHub codespaces, and they work well too.